.github/scripts/generate_binary_build_matrix.py

```diff
@@ -45,7 +45,7 @@
         "nvidia-curand-cu11==10.3.0.86; platform_system == 'Linux' and platform_machine == 'x86_64' | "
         "nvidia-cusolver-cu11==11.4.1.48; platform_system == 'Linux' and platform_machine == 'x86_64' | "
         "nvidia-cusparse-cu11==11.7.5.86; platform_system == 'Linux' and platform_machine == 'x86_64' | "
-        "nvidia-nccl-cu11==2.21.5; platform_system == 'Linux' and platform_machine == 'x86_64' | "
+        "nvidia-nccl-cu11==2.24.3; platform_system == 'Linux' and platform_machine == 'x86_64' | "
```

Comment:
Yeah, we can't do this until we fully complete the ManyLinux transition?

Reply:
Can I leave it pinned to the older version for 11.8?

```suggestion
        "nvidia-nccl-cu11==2.21.5; platform_system == 'Linux' and platform_machine == 'x86_64' | "
```